### PR TITLE
LayerManagerControl: Prevent conversion of filenames into lowercase

### PR DESCRIPTION
**Pull request by rajendraprasadkj**
Added check to prevent conversion of filename into lowercase

### CODE REVIEW

**Comment by efriedrich** [comment]
Hi Rajendra,
please remove the jira ticket id from the pull request and change the commit name to reflect what is fixed adn not the issue description, some thing like "don't convert dump filenames into lowercase, use it as it was provided"

**Comment by rajendraprasadkj** [comment]
Hi Eugen,
I have updated the commit name as suggested.

File: ivi-layermanagement-examples/LayerManagerControl/src/Expression.cpp [hunk 1]
```diff
@@ -215,6 +215,12 @@ ExpressionList Expression::getNextExpressionClosure(string text)
 
             if (expr->isVar())
             {
+                if((expr->mName).compare("<file>") &&
+                   (expr->mName).compare("<filename>"))
+                {
+                    transform(text.begin(), text.end(), text.begin(), ::tolower);
+                }
```
Comment (eucan) [inline ivi-layermanagement-examples/LayerManagerControl/src/Expression.cpp]: You can remove it outside of if statement. you are also calling transform() in else statement. Then, you don't need to call transform function in two different places

**Comment by eucan** [comment]
Your commit message is too long. Your commit description does not describe how did you solve the problem.

**Comment by rajendraprasadkj** [comment]
@eucan,
I have addressed the review comment and modified the commit message, description as suggested.

**Pull request merged**